android-client-sdk/src/main/java/com/devcycle/sdk/android/api/DVCApiClient.kt

```diff
@@ -10,7 +12,8 @@ internal class DVCApiClient {
     private val adapterBuilder: Retrofit.Builder = Retrofit.Builder()
         .addConverterFactory(JacksonConverterFactory.create(JSONMapper.mapper))
 
-    fun initialize(baseUrl: String): DVCApi {
+    fun initialize(baseUrl: String, context: Context): DVCApi {
```

Comment:
Is this the main public interface folks use with our SDK? will we need to make this a major change?

Reply:
nope, it's internal

Reply:
the whole class is internal

Reply:
it's used to get the system service `ConnectivityManager` instance. It has guards around it so context doesn't return a service it won't crash